crates/aptos/src/move_tool/mod.rs

```diff
@@ -159,26 +155,14 @@ pub fn init_move_dir(
             .as_path(),
     )?;
 
-    let mut dependencies = BTreeMap::new();
```

Comment:
Are we switching back to local dependencies? I thought we wanted to use the fullnode address from the profile?

Reply:
Yeah.. this doesn't seem to work yet, so went back to the framework-rev